nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol

```diff
@@ -103,7 +103,7 @@ contract PolicyManager is Upgradeable {
 
     mapping (bytes16 => Policy) public policies;
     mapping (address => NodeInfo) public nodes;
-    Range public minFeeRateRange;
+    Range public FeeRateRange;
```

Comment:
this shoud start from lower case

Reply:
changed all to lowercase 'f' except:  `setFeeRateRange` and `delegateGetFeeRateRange`

Reply:
But events start from uppercase :) so for lines 54 and 155 better to return back to `FeeRateRangeSet`

Reply:
haha i have so much to learn